mos_tests/nova/functions/common.py

```diff
@@ -0,0 +1,130 @@
+from time import sleep, time
+
+
+def get_inst_id(novaclient, inst_name):
```

Comment:
Probably we need to sync it with https://github.com/Mirantis/mos-integration-tests/tree/master/mos_tests/heat/functions